python/cuml/test/test_random_forest.py

```diff
@@ -244,22 +224,37 @@ def test_rf_regression_default(datatype, column_info, nrows):
 
 
 @pytest.mark.parametrize('datatype', [np.float32])
-@pytest.mark.parametrize('column_info', [unit_param([20, 10]),
-                         quality_param([200, 100]),
-                         stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(500), quality_param(5000),
-                         stress_param(500000)])
-def test_rf_classification_seed(datatype, column_info, nrows):
+def test_rf_classification_seed(dataset, datatype):
 
-    ncols, n_info = column_info
-    X, y = make_classification(n_samples=nrows, n_features=ncols,
-                               n_clusters_per_class=1, n_informative=n_info,
-                               random_state=0, n_classes=2)
+    X, y = dataset
     X = X.astype(datatype)
     y = y.astype(np.int32)
     X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=0.8,
                                                         random_state=0)
-    for i in range(20):
+
+    cuml_model = curfc()
+    cuml_model.fit(X_train, y_train)
+    fil_preds = cuml_model.predict(X_test, predict_model="GPU")
+    cu_preds = cuml_model.predict(X_test, predict_model="CPU")
```

Comment:
I think we can remove `predict_model="CPU"` from `test_rf_classification_default` , `test_rf_regression_default`, `test_rf_classification_sparse`  and `test_rf_regression_sparse`. We test the behavior of `predict_model="CPU"` wrt `predict_model="GPU"` and sklearn in the first two tests.